cognite/client/data_classes/time_series.py

```diff
@@ -75,43 +75,26 @@ def __init__(
         self.legacy_name = legacy_name
         self._cognite_client = cast("CogniteClient", cognite_client)
 
-    def plot(
```

Comment:
If removing these plot methods, that should be noted in the changelog and I also believe the docs mention some stuff about these methods.

Reply:
Yup, see #1049